apps/vs-code-designer/src/app/commands/generateDeploymentScripts/azureScriptWizard.ts

```diff
@@ -0,0 +1,252 @@
+      } else if (userChoice.label === 'Choose Different Folder') {
+        selectedPath = await selectWorkspaceFolder(context, placeHolder); // Assuming selectWorkspaceFolder is a function you have for folder selection
+      }
+
+      if (selectedPath) {
+        SourceControlPathListStep.setSourceControlPath(context, selectedPath);
+      }
+    }
+  }
+
+  public shouldPrompt(_context: IAzureScriptWizard): boolean {
+    return true;
+  }
+}
+
+// Define the ConfigureInitialLogicAppStep class
+class ConfigureInitialLogicAppStep extends AzureWizardPromptStep<IAzureScriptWizard> {
+  public async prompt(context: IAzureScriptWizard): Promise<void> {
+    context.enabled = true;
+  }
+
+  public shouldPrompt(context: IAzureScriptWizard): boolean {
+    return context.enabled === undefined;
+  }
+
+  public async getSubWizard(context: IAzureScriptWizard): Promise<IWizardOptions<IAzureScriptWizard> | undefined> {
+    const azurePromptSteps: AzureWizardPromptStep<IActionContext>[] = [];
+    const subscriptionPromptStep: AzureWizardPromptStep<IActionContext> | undefined =
+      await ext.azureAccountTreeItem.getSubscriptionPromptStep(context);
+    if (subscriptionPromptStep) {
+      azurePromptSteps.push(subscriptionPromptStep);
+    }
+
+    azurePromptSteps.push(new ResourceGroupListStep());
+    return { promptSteps: azurePromptSteps };
+  }
+}
+
+// Define the setLogicappName class
+export class setLogicappName extends AzureWizardPromptStep<IAzureScriptWizard> {
+  public hideStepCount = true;
+
+  public async prompt(context: IAzureScriptWizard): Promise<void> {
+    context.logicAppName = await context.ui.showInputBox({
+      placeHolder: localize('setLogicappName', 'logicappName'),
+      prompt: localize('logicappNamePrompt', 'Provide a unique name for the logic app'),
+    });
+  }
+
+  public shouldPrompt(context: IAzureScriptWizard): boolean {
+    return !context.logicAppName;
+  }
+}
+
+// Define the setStorageAccountName class
+export class setStorageAccountName extends AzureWizardPromptStep<IAzureScriptWizard> {
+  public hideStepCount = true;
+
+  public async prompt(context: IAzureScriptWizard): Promise<void> {
+    context.storageAccountName = await context.ui.showInputBox({
+      placeHolder: localize('setStorageAccountName', 'storageAccountName'),
+      prompt: localize('storageAccountNamePrompt', 'Provide a unique name for the storage account'),
+    });
+  }
+
+  public shouldPrompt(context: IAzureScriptWizard): boolean {
+    return !context.storageAccountName;
+  }
+}
+
+// Define the setAppPlantName class
+export class setAppPlantName extends AzureWizardPromptStep<IAzureScriptWizard> {
+  public hideStepCount = true;
+
+  public async prompt(context: IAzureScriptWizard): Promise<void> {
+    context.appServicePlan = await context.ui.showInputBox({
+      placeHolder: localize('setAppPlantName', 'appPlanName'),
+      prompt: localize('appPlanNamePrompt', 'Provide a unique name for the app service plan'),
```

Comment:
["App Service"](https://learn.microsoft.com/en-us/azure/app-service/overview-hosting-plans) is a service name and must be capitalized:

```suggestion
      prompt: localize('appPlanNamePrompt', 'Provide a unique name for the App Service plan.'),
```